app/api/views/briefs.py

```diff
@@ -354,6 +354,21 @@ def get_brief(brief_id):
             'name': domain.name
         })
 
+    blackout_date_start = None
+    blackout_date_end = None
+    blackout_date_live = None
+    blackout_dates = key_values_service.get_by_key('blackout_dates')
+    if blackout_dates:
+        blackout_date_start = (
+            pendulum.parse(blackout_dates['data']['startDate'], tz='Australia/Canberra').date()
+        ).strftime('%Y-%m-%d')
+        blackout_date_end = (
+            pendulum.parse(blackout_dates['data']['endDate'], tz='Australia/Canberra').date()
+        ).strftime('%Y-%m-%d')
+    blackout_period = dict()
+    blackout_period['startDate'] = blackout_date_start
+    blackout_period['endDate'] = blackout_date_end
```

Comment:
This block of code is very similar to what's been added in `brief_edit_business.py`.  Do you think we should move it to a function in `brief_business.py`?

Reply:
Yes, there was significant code duplication - I have placed the new functions in a separate module (I tried placing them in `brief_business.py`, but this caused issues when importing).